format/view-spec.md

```diff
@@ -101,6 +101,8 @@ A view version can have more than one representation. All representations for a
 
 View versions are immutable. Once a version is created, it cannot be changed. This means that representations for a version cannot be changed. If a view definition changes (or new representations are to be added), a new version must be created.
 
+There should be no more than one view representation per dialect.
```

Comment:
Dialect is not something that can be used here because it is defined in the SQL representation. There is no "dialect" for the view version and there could be multiple dialects from multiple SQL representations.

Reply:
I've removed this sentence as I just noticed that this is already being mentioned:

> A view version can have multiple SQL representations of different dialects, but only one SQL representation per dialect.